File: app/src/main/java/com/blog/demo/system/DisplayMetricsActivity.java
package com.blog.demo.system;

import android.app.Activity;
import android.os.Bundle;
import android.support.annotation.Nullable;
import android.util.DisplayMetrics;
import android.widget.TextView;

import com.blog.demo.LogTool;
import com.blog.demo.R;

public class DisplayMetricsActivity extends Activity {

    @Override
    protected void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);

        setContentView(R.layout.activity_text);

        TextView textView = findViewById(R.id.text_view);

        DisplayMetrics dm = new DisplayMetrics();
        getWindowManager().getDefaultDisplay().getMetrics(dm);

        textView.setText("widthPixels = " + dm.widthPixels + "\n"
                    + "heightPixels = " + dm.heightPixels + "\n"
                    + "densityDpi = " + dm.densityDpi + "\n"
                    + "density = " + dm.density);
    }
}

File: app/src/main/java/com/blog/demo/component/DisplayMetricsActivity.kt
package com.blog.demo.component

import android.app.Activity
import android.os.Bundle
import android.util.DisplayMetrics
import android.widget.TextView
import com.blog.demo.R

class DisplayMetricsActivity : Activity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)

        setContentView(R.layout.activity_text)

        val dm = DisplayMetrics()
        windowManager.defaultDisplay.getMetrics(dm)

        val textView: TextView = findViewById(R.id.text_view)
        textView.text = """
                widthPixels = ${dm.widthPixels}
                heightPixels = ${dm.heightPixels}
                densityDpi = ${dm.densityDpi}
                density = ${dm.density}
                """.trimIndent()
    }

}